plugin-name/includes/class-plugin-name-activator.php

```diff
@@ -22,14 +23,132 @@
 class Plugin_Name_Activator {
 
 	/**
-	 * Short Description. (use period)
+	 * The $_REQUEST during plugin activation.
 	 *
-	 * Long Description.
+	 * @since    1.0.0
+	 * @access   private
+	 * @var      array    $request    The $_REQUEST array during plugin activation.
+	 */
+	private static $request = array();
+
+	/**
+	 * The $_REQUEST['plugin'] during plugin activation.
+	 *
+	 * @since    1.0.0
+	 * @access   private
+	 * @var      string    $plugin    The $_REQUEST['plugin'] value during plugin activation.
+	 */
+	private static $plugin  = 'plugin-name/plugin-name.php';
+
+	/**
+	 * The $_REQUEST['action'] during plugin activation.
+	 *
+	 * @since    1.0.0
+	 * @access   private
+	 * @var      array    $action    The $_REQUEST[action] value during plugin activation.
+	 */
+	private static $action  = 'activate';
+
+	/**
+	 * Activate the plugin.
+	 *
+	 * Checks if the plugin was (safely) activated.
+	 * Place to add any custom action during plugin activation.
 	 *
 	 * @since    1.0.0
 	 */
 	public static function activate() {
 
+		if ( false === self::get_request()
+			|| false === self::validate_request( self::$plugin, self::$action )
+			|| false === self::check_caps()
+			|| ! check_admin_referer( 'activate-plugin_' . self::$request['plugin'] )
+		 ) {
```

Comment:
`Found precision alignment of 1 spaces (WordPress.WhiteSpace.PrecisionAlignment.Found)`
Literally no idea what that means.
I see nothing wrong with the code - it seems related to a tab vs space, but it would get caught by my local WPCS.

Reply:
Just hit space once and the error should go away.

Like I mentioned earlier, if your editor is setup correctly with the WP ruleset and PHPCS, this should be able to be fixed automatically.